ui/lib/core/addon/helpers/message-types.js

```diff
@@ -34,6 +35,9 @@ export const MESSAGE_TYPES = {
 };
 
 export function messageTypes([type]) {
+  if (!([type] in MESSAGE_TYPES)) {
```

Comment:
I don't think this should be in brackets? Since it's destructured as the arg

Reply:
Making a PR fix, thanks for pointing out!